security/advancedtls/crl_test.go

```diff
@@ -636,17 +641,33 @@ func setupTLSConn(t *testing.T) (net.Listener, *x509.Certificate, *ecdsa.Private
 // TestVerifyConnection will setup a client/server connection and check revocation in the real TLS dialer
 func TestVerifyConnection(t *testing.T) {
 	l, cert, key := setupTLSConn(t)
+
+	done := make(chan interface{}, 1)
```

Comment:
Typically a `done` channel used in this capacity can be unbuffered, and the signal used is closing it instead of pushing `true` to it.  Also `make(chan struct{})` is ever so slightly more optimal in that case.